XRTK-Core/Packages/com.xrtk.core/Services/DiagnosticsSystem/XRTK.Services.DiagnosticsSystem.asmdef

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "XRTK.Services.DiagnosticsSystem",
```

Comment:
Not sure we need an assembly definition for this service. 

What's the benefit of having it?

Reply:
It was there in MRTK so I kept it. I'll remove it.